rocAL/rocAL/include/pipeline/commons.h

```diff
@@ -81,6 +96,21 @@ enum class DecodeMode
     CPU = 1
 };
 
```

Comment:
@fiona-gladwin Can we keep this uniform and either "= 1" everywhere or remove it here?